hummingbot/connector/exchange/bittrex/bittrex_exchange.py

```diff
@@ -0,0 +1,403 @@
+        return CONSTANTS.EXCHANGE_INFO_PATH_URL
+
+    @property
+    def trading_pairs_request_path(self):
+        return CONSTANTS.EXCHANGE_INFO_PATH_URL
+
+    @property
+    def check_network_request_path(self):
+        return CONSTANTS.SERVER_TIME_URL
+
+    @property
+    def trading_pairs(self):
+        return self._trading_pairs
+
+    @property
+    def is_cancel_request_in_exchange_synchronous(self) -> bool:
+        return True
+
+    @property
+    def is_trading_required(self) -> bool:
+        return self._trading_required
+
+    def supported_order_types(self):
+        return [OrderType.LIMIT, OrderType.LIMIT_MAKER]
+
+    def _is_request_exception_related_to_time_synchronizer(self, request_exception: Exception):
+        return False
+
+    def _create_web_assistants_factory(self) -> WebAssistantsFactory:
+        return web_utils.build_api_factory(
+            time_synchronizer=self._time_synchronizer,
+            auth=self._auth)
+
+    def _create_order_book_data_source(self) -> OrderBookTrackerDataSource:
+        return BittrexAPIOrderBookDataSource(
+            trading_pairs=self._trading_pairs,
+            connector=self,
+            api_factory=self._web_assistants_factory)
+
+    def _create_user_stream_data_source(self) -> UserStreamTrackerDataSource:
+        return BittrexAPIUserStreamDataSource(auth=self._auth)
+
+    async def _place_order(self,
+                           order_id: str,
+                           trading_pair: str,
+                           amount: Decimal,
+                           trade_type: TradeType,
+                           order_type: OrderType,
+                           price: Decimal,
+                           **kwargs
+                           ) -> Tuple[str, float]:
+
+        path_url = CONSTANTS.ORDER_CREATION_URL
+        body = {
+            "marketSymbol": await self.exchange_symbol_associated_to_pair(trading_pair=trading_pair),
+            "direction": trade_type.name,
+            "type": "LIMIT",
+            "quantity": f"{amount:f}",
+            "limit": f"{price:f}",
+        }
+        if order_type is OrderType.LIMIT:
+            body.update({
+                "timeInForce": "GOOD_TIL_CANCELLED"
+            })
+        elif order_type is OrderType.LIMIT_MAKER:
+            body.update({
+                "timeInForce": "POST_ONLY_GOOD_TIL_CANCELLED"
+            })
+        body.update({
+            "clientOrderId": order_id
+        })
+        order_result = await self._api_post(
+            path_url=path_url,
+            data=body,
+            is_auth_required=True)
+        o_id = order_result["id"]
+        transact_time = _get_timestamp(order_result["createdAt"])
+        return o_id, transact_time
+
+    async def _place_cancel(self, order_id: str, tracked_order: InFlightOrder):
+        exchange_id = await tracked_order.get_exchange_order_id()
+        api_params = {
+            "orderId": exchange_id
+        }
```

Comment:
According to the documentation this DELETE endpoint does not require parameters (the order id is part of the URL)
https://bittrex.github.io/api/v3#operation--orders--orderId--delete

Reply:
Removed the params